### PR TITLE
chore: Added details of Asuncion which is capital of Paraguay

### PR DESCRIPTION
**Pull request by shriraj221094**
Adding the data of Asuncion which is the capital of [Paraguay](https://www.wikidata.org/wiki/Q2933). The city forms an autonomous [capital district](https://en.wikipedia.org/wiki/Asunci%C3%B3n), not a part of any department.

### CODE REVIEW

**Comment by shriraj221094** [comment]
@patrickdaze @fzzio @skrollme @bor0 Can someone please review this PR?

**Review by skrollme — APPROVED** [review]

**Comment by skrollme** [comment]
Looks good, @shriraj221094 but my approval has no weight in this project since I have no write-permission 🤷

**Comment by shriraj221094** [comment]
@dr5hn Could you please review this PR?

File: sql/world.sql [hunk 1]
```diff
@@ -156496,7 +156496,7 @@ INSERT INTO `states` (`id`, `name`, `country_id`, `country_code`, `fips_code`, `
 (5218, 'Navassa Island', 234, 'UM', NULL, '76', 'island', 18.41006890, -75.01146120, '2023-03-25 20:08:55', '2023-03-25 20:08:55', 1, 'Q25359'),
 (5219, 'Palmyra Atoll', 234, 'UM', NULL, '95', 'island', 5.88850260, -162.07866560, '2023-03-25 20:08:55', '2023-03-25 20:08:55', 1, 'Q123076'),
 (5220, 'Wake Island', 234, 'UM', NULL, '79', 'island', 19.27961900, 166.64993480, '2023-03-25 20:08:55', '2023-03-25 20:08:55', 1, 'Q43296');
```
Comment (dr5hn) [inline sql/world.sql:156498]: This line should end with `,`

**Review by dr5hn — CHANGES_REQUESTED** [review]
Please resolve my comments

File: sql/world.sql (hunk 1 above)
Comment (shriraj221094) [reply sql/world.sql:156498]: Updated the mentioned line. Could you please approve the PR?

**Review by dr5hn — APPROVED** [review]

**Pull request merged**